ros2topic/ros2topic/verb/echo.py

```diff
@@ -136,7 +142,8 @@ def cb(msg):
 def subscriber_cb_csv(truncate_length, noarr, nostr):
     def cb(msg):
         nonlocal truncate_length, noarr, nostr
-        print(message_to_csv(msg, truncate_length=truncate_length, no_arr=noarr, no_str=nostr))
+        print(message_to_csv(msg, truncate_length=truncate_length,
+                             no_arr=noarr, no_str=nostr))
```

Comment:
nitpick: For future, avoid changes unrelated to the PR. It helps keep the diff minimal, reducing chance for conflict with other changes (I ran into a conflict here).

Regarding style, this type of indentation is against our [style guide](https://index.ros.org/doc/ros2/Contributing/Code-Style-Language-Versions/#python) that prefers [hanging indentation](https://www.python.org/dev/peps/pep-0008/#indentation); although our linters don't catch it so I cannot blame anyone :man_shrugging: